be/package.json

```diff
@@ -8,13 +8,12 @@
     "dev:start": "nodemon",
     "debug": "DEBUG=express:* node dist/app.js",
     "test": "jest --passWithNoTests",
-    "lint": "eslint ./src",
+    "lint": "eslint --cache --fix ./src",
     "build": "tsc && cp -r src/static ./dist",
     "generate": "rm -R -f src/generated/* && openapi --input ../openapi.yaml --output ./src/generated",
     "generate:build": "npm run generate && npm run build",
     "generate:build:start": "npm run generate && npm run build && npm run start",
-    "build:start": "npm run build && npm run start",
-    "package": "npm run generate:build && copy-node-modules . ./dist"
```

Comment:
Please do not delete the `package`-script

Reply:
Restored